### PR TITLE
[3/X] Update feature/sql-refactor | CommandExecutors

### PR DESCRIPTION
**Pull request by MarkL4YG**
After having had very much to do for school, I am back to the sql-refactor :)  
Let's get started with some easier changes first.  
  
This PR updates the ``CommandExecutor`` classes with the reworked account classes.  
Aside from that I've simplified some code and applied some SonarLint suggestions in the executors where that was easily done without changing the code too much.
For that, I added a small convenience method to the ``TEAccountManager`` which should make ``#getOrCreateAccount`` calls way more readable.
  
Oh and: This PR includes the changes made to ``develop``.

### CODE REVIEW

File: src/main/java/com/erigitic/commands/AdminPayCommand.java [hunk 1]
```diff
@@ -81,18 +85,14 @@ public CommandResult execute(CommandSource src, CommandContext args) throws Comm
...
+                String invokerMessageId = amountStr.contains("-") ? "command.adminpay.remove.sender" : "command.adminpay.send.sender";
+                String recipientMessageId = amountStr.contains("-") ? "command.adminpay.remove.recipient" : "command.adminpay.send.recipient";
 
-                    if (recipient.isOnline()) {
-                        recipient.getPlayer().get().sendMessage(messageManager.getMessage("command.adminpay.remove.recipient", messageValues));
-                    }
+                src.sendMessage(messageManager.getMessage(invokerMessageId, messageValues));
+                if (recipient.isOnline()) {
+                    recipient.getPlayer()
+                            .orElseThrow(() -> new TERuntimeException("Recipient is only but is not a player!"))
```
Comment (Erigitic) [inline src/main/java/com/erigitic/commands/AdminPayCommand.java]: Error message is worded a bit weird here

```suggestion
                            .orElseThrow(() -> new TERuntimeException("Recipient is not a player!"))
```

File: src/main/java/com/erigitic/commands/BalanceTopCommand.java [hunk 2]
```diff
@@ -98,23 +95,27 @@ public CommandResult execute(CommandSource src, CommandContext args) throws Comm
...
-                while (set.next()) {
-                    BigDecimal amount = set.getBigDecimal(currencyColumn);
-                    UUID uuid = UUID.fromString(set.getString("uid"));
-                    Optional<User> optUser = Sponge.getServiceManager().provideUnchecked(UserStorageService.class).get(uuid);
-                    String username = optUser.map(User::getName).orElse("unknown");
-
-                    accountBalances.add(Text.of(TextColors.GRAY, username, ": ", TextColors.GOLD, currency.format(amount)));
+            String query = "SELECT * FROM balances WHERE `currency_ident` = '?' ORDER BY `balance` DESC LIMIT 10";
+            try (Connection connection = totalEconomy.getTESqlManager().getCommandConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
```
Comment (Erigitic) [inline src/main/java/com/erigitic/commands/BalanceTopCommand.java]: I think there is extra whitespace at the beginning of this line. That should be removed to be in line with the rest of the code.

File: src/main/java/com/erigitic/commands/SetBalanceCommand.java [hunk 3]
```diff
@@ -97,27 +99,24 @@ public CommandResult execute(CommandSource src, CommandContext args) throws Comm
     /**
      * Get the transaction result returned from setting a player's balance.
      *
-     * @param recipientAccount The account
-     * @param amount Amount of currency involved in the transaction
-     * @param optCurrencyName Currency name
+     * @param recipient    The account
```
Comment (Erigitic) [inline src/main/java/com/erigitic/commands/SetBalanceCommand.java]: Don't worry about lining up the descriptions of the params. One space after the param name will suffice.

File: src/main/java/com/erigitic/commands/BalanceTopCommand.java (hunk 2 above)
Comment (MarkL4YG) [reply src/main/java/com/erigitic/commands/BalanceTopCommand.java]: Uh, no. Not really.
That's how IntelliJ formats multi-line try-with-resources initializers.
I think the reason is that it is aligned with the first line this way. (Which also means that it is offset by one char due to the brace.

File: src/main/java/com/erigitic/commands/SetBalanceCommand.java (hunk 3 above)
Comment (MarkL4YG) [reply src/main/java/com/erigitic/commands/SetBalanceCommand.java]: Interesting. I don't even remember doing that ^^.  
Whatever - will change that anyways :D

**Pull request merged**